### DISCUSSION TITLE
How do I set HomeAssistant's proxy?

### DISCUSSION THREAD

**Discussion by MichaelOfGH** [home-assistant/operating-system#1763] [Q&A]
I use HASS in China, and there is no stable network linked to Github. But I have V2ray service. I want to make proxy work in HASS OS.

And my problem is , I stuck on prepare page...

![image](https://user-images.githubusercontent.com/28975359/155458667-0bdb4297-7ea3-4d4b-9c13-563d421bc66a.png)

![image](https://user-images.githubusercontent.com/28975359/155459414-df0ad297-833c-4b31-a915-2dab3455be2c.png)

**Comment by agners** [comment]
At this point there is no global proxy configuration. There is a feature request for that feature at https://community.home-assistant.io/t/support-http-proxy/204661.

**Comment by F0xss** [comment]
Russia connect to waiting problem
